docs/source/development/pack_testing.rst

```diff
@@ -0,0 +1,98 @@
+:orphan:
+
+Pack Testing
+============
+
+This section includes information on pack testing - where to put the tests,
+how to write the tests, mock classes which can be used to make testing
+easier, etc.
+
+Test File Locations and Names
+-----------------------------
+
+All the test files should go into ``<pack name>/tests/`` directory. If tests
+include any fixtures, they should be put in the ``<pack name>/tests/fixtures/``
+directory.
+
+Test files should follow the following naming conventions:
+
+* ``test_action_<action name>.py`` for action tests. For example, if the action
```

Comment:
The resource part in the name is a recommendation, but the `test_*.py` part is a requirement from a test runner.

This requirement can be relaxed of course, but I think it's a good one since all the Python tests tend to follow it :)